paper.tex

```diff
@@ -805,27 +809,45 @@ \section*{Inheritance and ancestral material}\label{Ancestry_resolution}
 joining \textsf{i} to its ancestor \textsf{k} then carries
 material on $(0, 2]$ since although \textsf{i} directly inherits interval
 $(0, 3]$ from \textsf{k} (Fig~\ref{fig-ancestry-resolution}B), only $(0, 2]$ is
-ancestral to the sample. This process is directly analogous to Hudson's
-simulation algorithm
+ancestral to the sample. For each node, we also keep track of the number of
+descendant samples over each genomic interval, and stop tracking that
+interval once the lineages from all samples have coalesced. For example,
+the edge from \textsf{k} to \textsf{n} omits the region $(0,2]$ because it has
+already fully coalesced (indicated by a fully black genomic region).
+This process is directly analogous to Hudson's simulation algorithm
 (Appendix XXX).
 
-These alternatives of recording the direct inheritance of genetic material from
-parent to child vs the propagation of genomic intervals ancestral to the sample
-are analogous to running forwards-in-time vs backwards-in-time simulations.
-Indeed, the process of taking a graph with edges annotated with direct parent-child
-inheritance (Fig~\ref{fig-ancestry-resolution}B) and producing the equivalent
-``ancestry resolved'' graph in which only those genomic intervals that are
-ancestral to the sample are retained (Fig~\ref{fig-ancestry-resolution}C)
-is the same key algorithm underlying recent advances in forwards-in-time simulation.
+In practice, the backwards-in-time Griffiths process that produces
+ARGs such as that shown in Figs~{fig-ancestry-resolution}A~and~B
+is rarely simulated, because the resulting ``big ARG'' (Appendix XXX) is
+dominated by nodes such as \textsf{g} and \textsf{j} which contain information
+about inheritance in non-ancestral material, as well as nodes such as
+\textsf{o} and \textsf{q} which track details of ancestry even once the
+local genomic region has fully coalesced. Instead, simulating
+ARGs from the coalescent with recombination is usually done by
+running Hudson's algorithm to produce a sample-resolved ARG directly.
+Moreover, when inferring ARGs from genetic data, there is no information
+in the sample set that can be used to resolve these nodes, so that they
```

Comment:
I would make this a bit stronger - they are entirely unknowable, because there is no information in the sample about them by definition.

Reply:
I think that's true, at least for g and f. But I'm not sure that e.g. the grand MRCA is entirely unknowable (I think you can prove it exists in the Griffiths model, even if you can't date it). These are simply unknowable from data on the samples. I guess there are other ways to know about non-ancestral nodes too. But I'm happy to rephrase - you think I should take out the "essentially"?

Reply:
I'm going to leave this as it is, and we can pick up in a further pass if necessary.